### PR TITLE
Prevent merging callouts, when deleting content in between callout nodes

### PR DESCRIPTION
**Pull request by whawker**
### Description

Given this scenario, pressing `backspace` would merge two callout nodes

Screenshot 2020-11-09 at 17 22 04

**Before**: 
Screenshot 2020-11-09 at 17 26 26

**After**:
Screenshot 2020-11-09 at 17 22 18

### Checklist

- [x] I have read the [**contributing**](https://github.com/remirror/remirror/blob/HEAD/docs/contributing.md) document.
- [x] My code follows the code style of this project and `pnpm fix` completed successfully.
- [ ] I have updated the documentation where necessary.
- [x] New code is unit tested and all current tests pass when running `pnpm test`.

### CODE REVIEW

**Comment by whawker** [comment]
Seems to be an issue with select all and delete, converting to draft in the meantime

**Comment by whawker** [comment]
Apologies, was just an issue with my own setup regarding extension priority

File: .changeset/warm-pugs-unite.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@remirror/extension-callout': minor
+'@remirror/playground': patch
```
Comment (ifiokjr) [inline .changeset/warm-pugs-unite.md:3]:
```suggestion
```

This changeset isn’t needed since it’s a private package.

**Review by ifiokjr — APPROVED** [review]
Looks really good 👍

**Comment by whawker** [comment]
Fixup'd the changeset :+1:

**Pull request merged**